### PR TITLE
Demo instance goal

### PR DESCRIPTION
**Pull request by nicksnyder**
The lack of a dedicated demo instance slowed down the Cloud team this month, so I am proposing adding this goal for the distribution team.

### CODE REVIEW

**Comment by slimsag** [comment]
I don’t think this needs to be a goal, that seems to imply this is a larger undertaking than it actually is - there is an issue open to do this and I expect to have it done as part of 3.21, if not sooner.

**Comment by pecigonzalo** [comment]
Looping the issue link in https://github.com/sourcegraph/sourcegraph/issues/13604

**Comment by nicksnyder** [comment]
Glad to see this is already an issue (I didn't know that). I will leave it up to the team on whether to also track this as a goal or not.

> I expect to have it done as part of 3.21, if not sooner.

Can we add that to the 3.21 milestone to communicate that (it is ok if it happens earlier).

File: handbook/engineering/distribution/goals.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 ## Medium-term goals
 
+### Create Sourcegraph demo instance
+
+- **Problem:** Customer engineering needs to demo Sourcegraph to customers. Many things can be demoed on Sourcegraph.com, but some things can't. For those things, CE has been using our dogfood instance sourcegraph.sgdev.org. Engineers need to use our dogfood instance for testing, which either puts demos at risk, or delays testing when demos happen.
+- **Plan:** Create demo.sourcegraph.com that CE uses for demos. This can be a small instance with a limited number of repos that is specifically designed for demo paths. sourcegraph.sgdev.org will continue to be our large scale dogfood instance for engineering testing.
+- **Owners:** TBD
+- **Status:** Not started
```
Comment (nicksnyder) [inline handbook/engineering/distribution/goals.md:12]:
```suggestion
- **Status:** https://github.com/sourcegraph/sourcegraph/issues/13604
```

**Comment by slimsag** [comment]
I guess my point was goals feels like it should be fairly high-level: a 15-30m task doesn't really fit that category in my view, otherwise most issues we work on should also become goals in this doc.

i digress, though, I'm happy to have this here on merit of "it feels big" and check that off as a win quickly.

**Comment by slimsag** [comment]
Perhaps one way to rephrase this is that it has always been a goal, just not documented, and that managed instances has enabled us to do this in a way where we treat the CE/Sales team as a "customer" of Sourcegraph?

File: handbook/engineering/distribution/goals.md [hunk 2]
```diff
@@ -4,6 +4,13 @@ Goals are continuously updated and reviewed. If you find these goals do not refl
 
 ## Medium-term goals
 
+### Create Sourcegraph demo instance
+
+- **Problem:** Customer engineering needs to demo Sourcegraph to customers. Many things can be demoed on Sourcegraph.com, but some things can't. For those things, CE has been using our dogfood instance sourcegraph.sgdev.org. Engineers need to use our dogfood instance for testing, which either puts demos at risk, or delays testing when demos happen.
+- **Plan:** Create demo.sourcegraph.com that CE uses for demos. This can be a small instance with a limited number of repos that is specifically designed for demo paths. sourcegraph.sgdev.org will continue to be our large scale dogfood instance for engineering testing.
+- **Owners:** TBD
```
Comment (pecigonzalo) [inline handbook/engineering/distribution/goals.md:11]:
```suggestion
- **Owners:** Robert, Gonza
```

**Review by pecigonzalo — APPROVED** [review]
I believe that is the case, we are also working on this because `manage-instances` code enabled this to be done fairly easily and the work on dogfooding has enabled that cluster to work as intended, upgrading more frequently, which is a problem for the way it was used before for demos.

**Comment by christinaforney** [comment]
It looks like this will be added as a goal. Perhaps as a later follow up after complete is to clarify the infrastructure area of ownership for the distribution team to be more specific about the instances maintained. For example:

- Internal Sourcegraph instances, including:
   1. A demo instance that is kept up-to-date and is stable for CE and Sales to use
   1. A dogfood instance for the engineering team to try out features in a Server environment (vs. Cloud)

**Review by christinaforney — APPROVED** [review]

**Comment by nicksnyder** [comment]
we have a demo instance now, so closing this

**Pull request closed without merging**